MAUI/DataGrid/row-drag-and-drop.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: post
+title: Row Drag and Drop in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Row Drag and Drop support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .net maui datagrid, .net maui grid, .net grid maui, .net maui row drag and drop, maui row drag and drop
+---
+
+# Row Drag and Drop in MAUI DataGrid (SfDataGrid)
+
+The `SfDataGrid` control allows you to drag and drop rows by setting the `SfDataGrid.AllowDraggingRow` property to `true`. Dragging is enabled by long pressing and dragging a row on mobile platforms and by clicking and dragging on desktop platforms. A row drag view is displayed during the drag operation. You can handle drag-and-drop operations based on your requirements by using the `SfDataGrid.QueryRowDragging` event.
+
+To enable row drag and drop functionality, please follow the code example below:
+
+{% tabs %}
+{% highlight XAML %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       ItemsSource="{Binding OrderInfoCollection}"  
+                       AllowDraggingRow="True">
+ </syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+dataGrid.AllowDraggingRow = true;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="DataGrid row drag and drop" src="Images\row-drag-drop\maui-datagrid-dragging.gif" width="404"/>
+
+## Dragging scenarios
+
+The SfDataGrid supports drag-and-drop operations for both data rows and groups in the following scenarios:
+
+Records can be reordered to any position using auto-scrolling.
+Group positions can be reordered using drag and drop; however, groups cannot be nested within other groups.
+Data rows can be reordered within the same group or moved to different groups.
+
+N> Reordering changes are made only in the `SfDataGrid.View`, not in the underlying data. As a result, these changes will be reverted when performing sorting, grouping, or any other operation that refreshes the view. To persist reordering changes in the underlying data, handle the `QueryRowDragging` event as explained below in [Reordering underlying data](#Reordering underlying data).
+
+## Row drag-and-drop template
+
+The SfDataGrid allows you to load specific content during row drag-and-drop operations using the `SfDataGrid.RowDragDropTemplate`. This template can be defined either in code or XAML.
+
+## Default template
+
+The default template will be used for row drag-and-drop operations unless another template is explicitly assigned.
+
+<img alt="DataGrid row drag and drop" src="Images\row-drag-drop\maui-datagrid-dragging.gif" width="404"/>
+
+## Customizing row drag-and-drop template
```

Comment:
It should be heading level 3.

      ### Customizing row drag-and-drop template